backend/compilers/gcc2.6.3-mipsel/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "platform": "ps1",
+    "cc": "mips-linux-gnu-cpp -Wall -lang-c -gstabs \"$INPUT\" | \"${COMPILER_DIR}\"/cc1 -mips1 -mcpu=3000 -G0 -fno-builtin $COMPILER_FLAGS | mips-linux-gnu-as -march=r3000 -mtune=r3000 -no-pad-sections -O1 -o \"$OUTPUT\""
```

Comment:
Think `-G0 -fno-builtin` should be exposed as flags rather than hardcoded

Reply:
Also wonder if the `-O1` will become a sticking point. @ethteck we briefly talked about exposing assembler flags but it's nontrivial I guess..

Reply:
yeah it's a whole other field for the model + api so I agree it's useful but ...it work